message/validation/timeLocation.go

```diff
@@ -0,0 +1,41 @@
+package validation
+
+import (
+	"strings"
+
+	"github.com/xmidt-org/glaukos/message"
+)
+
+// TimeLocation is an enum to determine what should be used in timeElapsed calculations
+type TimeLocation int
+
+const (
+	Birthdate TimeLocation = iota
+	Boottime
+)
+
+var (
+	timeLocationUnmarshal = map[string]TimeLocation{
+		"birthdate": Birthdate,
+		"boot-time": Boottime,
+	}
+)
+
```

Comment:
Where are these two functions being used? Or is this prep for future work?

Reply:
This is prep for future work